spec/lib/iban_client/iban_spec.rb

```diff
@@ -57,6 +57,15 @@
       let(:iban) { 'invalid_json' }
       it { expect { subject.bic }.to raise_error(IbanClient::RequestError) }
     end
+
+    context 'with an unexpected error' do
+      before do
+        allow(RestClient::Request).to receive(:execute).and_raise(StandardError, "Cant connect")
+      end
+      it { expect { subject.bic }.to raise_error(IbanClient::RequestError) do |error|
```

Comment:
Instead of testing the internal of the error (the fact it has an attribute named `response`), we can just test the message:
```ruby
it { expect { subject.bic }.to raise_error(IbanClient::RequestError, /Response: Cant connect/) }
```